actions/game_server_MOD.js

```diff
@@ -414,11 +414,12 @@ module.exports = {
     const port = this.evalMessage(data.serverport, cache);
 
     const Mods = this.getMods();
```

Comment:
```suggestion
```

Reply:
this can be removed since its not being used anymore., I'll leave that up to you though.

Reply:
Removed that now